tests/Unit/Domain/Test_TagsTimeDependent.cpp

```diff
@@ -38,8 +38,6 @@
 namespace {
 template <size_t Dim>
 void test_tags() noexcept {
-  TestHelpers::db::test_simple_tag<domain::Tags::InitialFunctionsOfTime<Dim>>(
-      "InitialFunctionsOfTime");
   TestHelpers::db::test_simple_tag<domain::Tags::FunctionsOfTime>(
```

Comment:
please remove this test as well; it is duplicated in tests/Unit/Domain/FunctionsOfTime/Test_Tags.cpp